jujugui/static/gui/src/app/components/user-profile/create-model-button/create-model-button.js

```diff
@@ -95,7 +95,7 @@ YUI.add('create-model-button', function() {
       @method _nextCreateStep
     */
     _nextCreateStep: function() {
-      if (this.props.gisf) {
+      if (this.props.gisf || (window.flags && window.flags.blues)) {
```

Comment:
Lets remove `gisf` from this check entirely as it is no longer a condition for this functionality. It will become the default for all cases with the removal of the `blues` flag. Maybe actually a XXX outlining this would be helpful here.